uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -378,6 +378,29 @@ _Response Fields_
   Added in UAA 3.3.0
 </aside>
 
+When an access token is revoked using this endpoint, the UAA Introspect Token endpoint (``/introspect``)
+will respond with ``"active": false`` when presented with the revoked access token.
+
+If the access token is in the JWT format (as opposed to the opaque format), the server config ``uaa.jwt.revocable`` or
+the Identity Zone config ``config.tokenPolicy.jwtRevocable`` must be set to ``true`` for
+this feature to work. However, OAuth resource servers are generally not required to call the UAA Introspect
+Token endpoint to validate the status of the token. Once issued, a valid access token
+in the JWT format is generally considered valid until its expiry. Hence, we do not recommend
```

Comment:
> is generally considered valid until its expiry

IMO the "generally" is confusing. I would phrase it:

> A JWT token should be considered valid and irrevocable until its expiry.

Reply:
Also, I would like us to be more clear and specific and add something like

> Even-though the `/revoke` endpoint may return a 200 OK when calling it with a JWT token, a JWT token should be considered valid and irrevocable until its expiry, because resource owners may not know that the token was revoked, and hence still grant access to the resource to the client.

Reply:
Point discussed and/or edits made.